static/docs/user-guide/dvcignore.md

```diff
@@ -18,26 +19,126 @@ provide similar functionality as `.gitignore` files provide for `git`.
 - Each line should contain only one pattern;
```

Comment:
please, use `.` instead of `;` at the end of each item

Reply:
Ok, but I did it to match changes introduced for unification of bullet lists. Isn't it the way we are handling bullet lists now?

Reply:
Nope, we're avoiding `;` moving forward (: Refer to https://github.com/iterative/dvc.org/issues/491#issuecomment-514118621